components/lists/Friends.tsx

```diff
@@ -1,44 +1,44 @@
-import { FlatList, SafeAreaView, StyleSheet, View } from "react-native"
+import { ScrollView, StyleSheet, View } from "react-native"
 import CustomText from "@/components/ui/CustomText"
 import FriendCard from "@/components/Friend"
-import { visibleFriends } from "@/data/friends"
 import { useState } from "react"
+import { defaultUsers } from "@/data/users"
 
 export default function FriendsList() {
-  const [friendsList, setfriendsList] = useState(visibleFriends)
+  const [friendsList, setFriendsList] = useState(defaultUsers)
   const updateFriendsList = (userId: string) => {
-    setfriendsList((prevList) =>
+    setFriendsList((prevList) =>
       prevList.map((user) =>
         user.id === userId ? { ...user, selected: !user.selected } : user,
       ),
     )
   }
   return (
     <View style={styles.container}>
-      <CustomText size="sm" fontWeight="medium">
-        Who can see
+      <CustomText size="sm" fontWeight="normal" fontFamily="suisse">
+        Who can see it
       </CustomText>
-      <SafeAreaView>
-        <FlatList
-          data={friendsList}
-          renderItem={({ item }) => (
-            <FriendCard
-              handleChange={() => updateFriendsList(item.id)}
-              user={item}
-            />
-          )}
-          keyExtractor={(item) => item.id.toString()}
-        />
-      </SafeAreaView>
+      <ScrollView
```

Comment:
why did you change this to a  `ScrollView`? we can't have 2 scrollviews with the same orientation under each other.